falco.yaml

```diff
@@ -25,7 +25,7 @@
 #
 #     (Falco command-line arguments)
 #     (Falco environment variables)
-# Falco config files
+# Falco configs files
```

Comment:
Am not sure though, since we have `watch_config_files` too, and indeed we have multiple configuration (singular) files (plural), so `config_files` is not that bad after all, even if it somehow clashes with `rules_files` (but in this case, we have multiple files with multiple rules inside).
WDYT? Ie i am proposing to rename `configs_files` to `config_files`.
cc @leogr

Reply:
So we keep `config` (singular form)? Works for me. Could we re-audit if we are consistent everywhere?

Reply:
Ok Leo is on board with changing `configs_files` to `config_files`. I will add a new commit for that.